src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -147,5 +148,46 @@ export class SbUserComponent implements OnInit {
             height: 'auto'
         });
     }
-}
 
+  blockUser(user: User) {
+    this.deleteUserDialog = true;
+    this.user = user;
+  }
+  confirmBlock() {
+    const payload = {
+      "request": {
+        "userId": this.user.userId
+      }
+    }   
+    this.userService.blockUser(payload).subscribe(response => {
```

Comment:
merge block and unblock into one function

Reply:
resolved